simulariumio/data_objects/unit_data.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from pint import UnitRegistry
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class UnitData:
+    magnitude: float
+    name: str
+
+    def __init__(
+        self,
+        name: str,
+        magnitude: float = 1.0,
+    ):
+        """
+        This object contains data for units
+
+        Parameters
+        ----------
+        name: str
+            unit name for values (we support this list
+            https://github.com/hgrecco/pint/blob/master/pint/default_en.txt)
+        magnitude: float (optional)
+            multiplier for values (in case they are not given in whole units)
+            Default: 1.0
+        """
+        # standardize and simplify units with pint
+        ureg = UnitRegistry()
+        self._quantity = magnitude * ureg(name)
```

Comment:
What happens here if a non-valid unit is passed in? Like `"kitties"`

Reply:
it raises a pint.errors.UndefinedUnitError. I tried to add a test for that but couldn't get it to work

Reply:
I think it would be good to display an error message with a link to the list of acceptable units, but if others don't think it's necessary either I don't feel strongly about it :)